File: php/getting-localized-content/GetLocalizedItems.php
<?php
// DocSection: getting_localized_content_language
// Tip: Find more about PHP SDKs at https://docs.kontent.ai/php
// Defined by Composer to include required libraries
require __DIR__ . '/vendor/autoload.php';

use Kentico\Kontent\Delivery\DeliveryClient;
use Kentico\Kontent\Delivery\QueryParams;

$client = new DeliveryClient('8d20758c-d74c-4f59-ae04-ee928c0816b7');

$items = client->getItem('about_us', (new QueryParams())
            ->language('es-ES'));
// EndDocSection
?>

File: java/getting-localized-content/GetLocalizedItems.java
// DocSection: getting_localized_content_language
// Tip: Find more about Java SDK at https://docs.kontent.ai/java
import kentico.kontent.delivery.*;

DeliveryClient client = new DeliveryClient("8d20758c-d74c-4f59-ae04-ee928c0816b7");

List<NameValuePair> params = DeliveryParameterBuilder.params().language("es-ES").build();

// Create strongly typed models according to https://docs.kontent.ai/strongly-typed-models
CompletionStage<ArticleItem> item = client.getItem("about_us", ArticleItem.class, params);
// EndDocSection